crates/biome_js_analyze/tests/specs/nursery/useAnchorHref/invalid-2.jsx

```diff
@@ -0,0 +1 @@
+<a target="_blank">External</a> 
```

Comment:
There's one case that you didn't cover:

```jsx
<a target="_blank" {...props}>External</a> 
```

In this case, we **shouldn't trigger the rule**, because `props` might contain `href`.